src/utils/DefaultValueContext.h

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
```

Comment:
Move it under src/storage/context?

Reply:
`codec` use it, so I don't think it's better to move to Storage.